RecoEgamma/EgammaPhotonProducers/src/PhotonProducer.cc

```diff
@@ -359,6 +359,19 @@ void PhotonProducer::fillPhotonCollection(edm::Event& evt,
     float sigmaIetaIeta = sqrt(locCov[0]);
     float r9 =e3x3/(scRef->rawEnergy());
 
+    float full5x5_maxXtal =   noZS::EcalClusterTools::eMax( *(scRef->seed()), &(*hits) );
```

Comment:
Before dereferencing hits, please check it is non-null. There is a possibility that hits would equal 0.